.github/workflows/end-to-end.yml

```diff
@@ -222,27 +209,14 @@ jobs:
       with:
         args: route53 change-resource-record-sets --hosted-zone-id Z2STJRQSJO5PZ0 --change-batch file://apps/pipeline/config/exoscale/dns-${{ matrix.cluster }}.json
 
-  buckets-cleanup:
-    needs: apps
-    if: always()
-    runs-on: ubuntu-20.04
-    steps:
-    - name: Checkout bucket-manager
-      uses: actions/checkout@v2
-      with:
-        repository: elastisys/bucket-manager
-        ref: main
-        token: ${{ secrets.CK8S_PIPELINE_GITHUB_TOKEN }}
     - name: Delete buckets
+      if: matrix.cluster == 'sc'
       env:
         EXOSCALE_API_KEY: ${{ secrets.EXOSCALE_API_KEY }}
         EXOSCALE_API_SECRET: ${{ secrets.EXOSCALE_API_SECRET }}
       run: |
-        python3 -m venv venv
-        source venv/bin/activate
-        pip install -r S3/requirements.txt
-        ./S3/gen-s3cfg.sh exoscale "${EXOSCALE_API_KEY}" "${EXOSCALE_API_SECRET}" sos-ch-gva-2.exo.io > s3cfg.ini
-        ./S3/manager.sh --s3cfg s3cfg.ini --delete pipeline-exoscale-harbor pipeline-exoscale-velero pipeline-exoscale-es-backup pipeline-exoscale-influxdb pipeline-exoscale-sc-logs
+        ./apps/scripts/S3/generate-s3cfg.sh exoscale "${EXOSCALE_API_KEY}" "${EXOSCALE_API_SECRET}" sos-ch-gva-2.exo.io ch-gva-2 > s3cfg.ini
```

Comment:
Is this step needed since it's the same step as the bucket create?

Reply:
Your right it shouldn't be needed.